images/TEMPLATE/tests/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+// Invoke a script with the test.
+// $IMAGE_NAME is populated with the image name by digest.
+// TODO: Update or remove this test as appropriate.
+data "oci_exec_test" "manifest" {
+  digest      = var.digest
+  script      = "./EXAMPLE_TEST.sh"
+  working_dir = path.module
+}
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "helm" {
+  name             = "{{.ModuleName}}-${random_pet.suffix.id}"
+  namespace        = "{{.ModuleName}}-${random_pet.suffix.id}"
+  repository       = "" // TODO
+  chart            = "" // TODO
+  create_namespace = true
+
+  values = [
+    jsonencode({
+      // TODO:
+    })
+  ]
```

Comment:
Is there a reason why this would be preferred over `set{}`?

Reply:
I don't really know, I mostly just copied this from another helm_release and slapped a TODO on it.

If `set` is easier we can make that the default too, very easily.

Reply:
Mostly curious, but I don't think any change to this is warranted.